testing/test_skipping.py

```diff
@@ -382,7 +382,64 @@ def test_func():
         ])
 
 
-class TestSkipif:
+class TestSkip(object):
+    def test_skip_class(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip
+            class TestSomething(object):
+                def test_foo(self):
+                    pass
+                def test_bar(self):
+                    pass
+
+            def test_baz():
+                pass
+        """)
+        rec = testdir.inline_run()
+        rec.assertoutcome(skipped=2, passed=1)
+
+    def test_skip_no_reason(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip
+            def test_foo():
+                pass
+        """)
+        rec = testdir.inline_run()
+        rec.assertoutcome(skipped=1)
```

Comment:
I assume the reason in this case was `"False"`, right? I mean, the only possible parameter for `skip` is a reason, differently from `skipif` which receives `(condition, reason)`.